middleware/http_handler.go

```diff
@@ -33,19 +32,20 @@ func (h httpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	// create Span using SpanContext if found
 	sp := h.tracer.StartSpan(
-		h.name, kind.Server,
+		h.name,
+		zipkin.Kind(model.Server),
 		zipkin.Parent(sc),
 		zipkin.RemoteEndpoint(zipkin.NewEndpointOrNil("", r.RemoteAddr)),
 	)
 	defer sp.Finish()
 
 	// add our span to context
-	ctx := span.NewContext(r.Context(), sp)
+	ctx := zipkin.NewContext(r.Context(), sp)
 
 	// tag typical HTTP request items
 	zipkin.TagHTTPMethod.Set(sp, r.Method)
 	zipkin.TagHTTPUrl.Set(sp, r.URL.String())
-	zipkin.TagHTTPRequestSize.Set(sp, fmt.Sprintf("%d", r.ContentLength))
+	zipkin.TagHTTPRequestSize.Set(sp, strconv.FormatInt(r.ContentLength, 10))
```

Comment:
What about wrapping `10` with a constant describing what is it?

Reply:
I think given strconv.FormatInt being a standard library function that is a bit much...